sdk/python/testvectors/nem/account_key_link.py

```diff
@@ -1,15 +1,20 @@
-recipes = {
-	'schema_name': 'AccountKeyLinkTransactionV1',
-	'descriptors': [
-		{
+SCHEMA_NAME = 'AccountKeyLinkTransactionV1'
```

Comment:
as mentioned, would probably revert these and allow export to be array for Block / multiversion cases